### PR TITLE
Including documentation for the SHOULD_STORE, WRITE_CONCERN and REPLICA_SET configuration parameters

### PR DESCRIPTION
**Pull request by gtorodelvalle**
- 100% tests passed
- Implements https://github.com/telefonicaid/IoT-STH/issues/66
- Assigned to @frbattid

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -289,6 +289,8 @@ a counter used as the suffix for the log file name. Optional. Default value: und
 - COLLECTION_PREFIX: The prefix to be added to the collections in the databases. More information below. Optional. Default value: "sth".
 - DEFAULT_SERVICE_PATH: The service path to be used if not sent by the Orion Context Broker in the notifications. Optional. Default value: "/".
 - POOL_SIZE: The default MongoDB pool size of database connections. Optional. Default value: "5".
+- WRITE_CONCERN: The <a href="http://docs.mongodb.org/manual/core/write-concern/" target="_blank">write concern policy</a> to apply when writing data to the MongoDB database. Default value: "1".
+- SHOULD_STORE: Flag indicating if the raw and/or aggregated data should be persisted. Valid values are: "only-raw", "only-aggregated" and "both". Default value: "both".
```
Comment (vgarciag) [inline README.md]: Have you forgot the `REPLICA_SET` parameter?

Comment (gtorodelvalle) [inline README.md]: Indeed ;) Thanks for noticing! Including it :)

**Comment by gtorodelvalle** [comment]
Included the description of the REPLICA_SET configuration parameter ;) Thanks @vgarciag ! :)

**Comment by frbattid** [comment]
LGTM

**Pull request merged**